src/sentry/lang/java/processing.py

```diff
@@ -23,3 +35,206 @@ def deobfuscate_exception_value(data):
         )
 
     return data
+
+
+def _merge_frame_context(new_frame: dict[str, Any], symbolicated: dict[str, Any]):
```

Comment:
There is no comment in any of the codes. Some might be obvious, some might not. Can you add comments to make the code more maintainable?